project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@ng-rspack/source",
```

Comment:
Renamed the root project to not include @

Reply:
Is this needed?

Reply:
The reason is I have a small helper that maps project name to location.
I can also implement more logic and use the project graph and keep the name.

Let me try that!

Reply:
It is anyway not tracked ATM so I reverted it and will come later with a better solution